lib/server/routes/buckets.js

```diff
@@ -1804,6 +1802,8 @@ BucketsRouter.prototype.finishUpload = async function (req, res, next) {
 
     await Promise.all(bucketEntryShardsToCreate);
 
+    const savedBucketEntryObject = savedBucketEntry.toObject();
+
     res.status(200).send(savedBucketEntryObject);
```

Comment:
We need to return the saved bucket entry + the `_id` as `id`:

`{ ...savedBucketEntryObject, id: savedBucketEntryId } `

Reply:
I just checked the `savedBucketEntryObject` already has the  _id as `id`. However it would probably be safer to enforce it as you just described.

Shall I proceed with your suggested implementation?

Reply:
Yes, take into count that `toObject()` deletes the property `_id` from the object, so we can not rely on that

Reply:
Should be fixed now